metadata.rb

```diff
@@ -24,7 +24,7 @@
 depends          'apt' # We recommend '>= 2.1.0'. See CHANGELOG.md for details
 depends          'chef_handler', '~> 1.1' # We recommend '~> 1.3' with Chef < 12. See CHANGELOG.md for details
 depends          'windows' # We recommend '< 1.39.0' if running Chef >= 12.6. See README.md for details
-depends          'yum'
+depends          'yum', '~> 3.0'
```

Comment:
Let's use an optimistic version constraint here (`>= 3.0`), I've tested that `yum` `4.0` works well with our cookbook.

Since yum `4.0` is Chef 12+ only, we should add a comment on that though, i.e.:

```ruby
depends          'yum', '>= 3.0' # Use '~> 3.0' with Chef < 12
```